### DISCUSSION TITLE
how to i setup ipv6 on my docker-compose

### DISCUSSION THREAD

**Discussion by balajeek** [wg-easy/wg-easy#483] [General]
I switched from verizon to t-mobile today and noticed i was not able to access data when my wireguard is turned on.  when i turn off wireguard i was able to open safari and browse thru websites fine, but when i turn on wireguard, none of the page works on safari, but surprisingly some apps are able to connect and work, like my home assistant (configured with local IP), music and other some apps i tried.

Not sure what's going on, as i was ready on reddit about wireguard and t-mobile does not work well because t-mobile uses ipv6 or something.

So how can i get this setup so i can use wireguard while i am on cell network t-mobile.

**Comment by pheiduck** [comment]
We haven't implemented a feature to use IPv6 there is a PR: https://github.com/wg-easy/wg-easy/pull/745 still in WIP due untested.
